implementation/src/main/java/io/smallrye/mutiny/Uni.java

```diff
@@ -805,6 +805,17 @@ static UniCombine combine() {
         return onItem().transform(ignore -> null);
     }
 
+    /**
+     * Ignore the item emitted by this {@link Uni} and replace it with {@code null} and type {@link Void}.
+     * <p>
+     * This is a shortcut for {@code uni.onItem().transform(ignored -> null)}.
+     * 
```

Comment:
I would add a note explaining the difference with replaceWithNull.

Reply:
Agreed